### DISCUSSION TITLE
Hyde cli issue

### DISCUSSION THREAD

**Discussion by rubiin** [prasanthrangan/hyprdots#1283] [General]
When running any command on Hyde cli , i get the following error.
Hyperdots cli was working fine

![image](https://github.com/prasanthrangan/hyprdots/assets/8222059/7b22e2d5-96c7-4b33-93b0-61d0bc06c62a)

**Comment by kRHYME7** [comment]
Yow, Lemme fix it. The testing branch is ow merge so we need to update the CLI to point again to main.

**Reply by kRHYME7** [reply]
please update ``` yay -Sy hyde-cli-git ```
